### PR TITLE
Pillar page: UNS

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/unified-namespace/index.njk [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+meta:
+  title: "Unified Namespace (UNS)"
+  faq:
+  - question: "What is the UNS?"
+    answer: >
+      The unified namespace is a new data architecture to accellerate digital transformation. All data, from sensors, systems, and machines are connected to make their data available to a unified network of data producers and consumers. This speeds up
+      application building, system integration, and delivers faster insights into productivity.
+  - question: "Why is the UNS important?"
+    answer: >
+      The UNS makes data available for more people to make better data-driven decisions. Furthermore,
```
Comment (ZJvandeWeg) [inline src/unified-namespace/index.njk:13]:
```suggestion
      The UNS makes data available for more people to make better data-driven decisions. Furthermore,  it allows for quick integration of new services and machines.
```

File: src/unified-namespace/index.njk [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="text-white">
+        {% include "components/dividers/light-grey-curve-inverted.svg" %}
+    </div>
+    <!-- Frequently Asked Questions -->
```
Comment (Yndira-FlowForge) [inline src/unified-namespace/index.njk:158]: This section is not showing the answers, you need the script that's in the Node-RED page to make the accordions work.

**Review by Yndira-FlowForge — CHANGES_REQUESTED** [review]

**Comment by Yndira-FlowForge** [comment]
>  I'd rather merge sooner rather than later and update icons in a follow up PR next week. WDYT?

 I agree.
 
 I noticed you didn't add it to the navbar, is it because you want to wait until is fully ready?

**Review by Yndira-FlowForge — APPROVED** [review]

File: src/unified-namespace/index.njk (hunk 2 above)
Comment (Yndira-FlowForge) [reply src/unified-namespace/index.njk:158]: I created a partial an added it to the node red page and to this one as well

**Pull request merged**